gulpfile.js

```diff
@@ -21,6 +21,8 @@ var gulp = require('gulp'),
     connectModRewrite = require('connect-modrewrite'),
     ngConfig = require('gulp-ng-config'),
     prettyError = require('gulp-prettyerror'),
+    del = require('del'),
```

Comment:
@anweshknayak : we are already using del in gulp at line 17. So remove this line please.